### PR TITLE
feat: render skeleton of repo with a jsonnet lib

### PR DESCRIPTION
**Pull request by Duologic**
Rendered from Jsonnet:

- config.yml for k8s-gen
- LICENSE
- root/docs READMEs
- [mkdocs](https://mkdocs.org/) config and github action + requirements for the action

Removed the rendered files from the source to avoid confusion.

This depends on a change to `docsonnet` to render the docs in a directory tree instead of a flat file
structure: https://github.com/jsonnet-libs/docsonnet/pull/20

After this, rendering a new Jsonnet lib & docs from a CRD would look like this:

```
local config = import '../../jsonnet/config.jsonnet';

config.new(
  name='cert-manager',
  specs=[
    {
      output: '1.3',
      openapi: 'http://localhost:8001/openapi/v2',
      prefix: '^io\\.cert-manager\\..*',
      crd: 'https://github.com/jetstack/cert-manager/releases/download/v1.3.1/cert-manager.crds.yaml',
      localName: 'cert_manager',
    },
  ]
)
```

Next PR will automate the push to the remote repository.

### CODE REVIEW

File: .github/workflow/main.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+on: [push]
+
+jobs:
+  generate_job:
+    name: 'Generate Jsonnet libraries and their docs'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: make run
```
Comment (Duologic) [inline .github/workflow/main.yml:9]: Bah, to test this I need to merge to master first. :angry:

File: jsonnet/config.jsonnet [hunk 2]
```diff
@@ -111,13 +111,16 @@
...
+              with: {
+                'fetch-depth': 0,
+              },
             },
+            { uses: 'actions/setup-python@v2' },
+            { run: 'pip install -r requirements.txt' },
+            { run: "git config user.name 'github-actions[bot]' && git config user.email 'github-actions[bot]@users.noreply.github.com'" },
             {
-              name: 'Deploy docs',
-              uses: 'mhausenblas/mkdocs-deploy-gh-pages@master',
```
Comment (Duologic) [inline jsonnet/config.jsonnet]: This upstream action gave inconsistent results, second build couldn't push for some reason.

**Comment by Duologic** [comment]
I have tested this on https://github.com/jsonnet-libs/cert-manager-lib.

**Comment by Duologic** [comment]
@xvzf and @julienduchesne also interested in reviewing this?

**Review by xvzf — APPROVED** [review]
Generally those changes lgtm! 👍🏿

**Review by julienduchesne — APPROVED** [review]
👍

**Pull request merged**